src/algebra/char_zero.lean

```diff
@@ -111,6 +111,48 @@ end
 
 end nat
 
+namespace int
+variables {R : Type*} [comm_ring R] [char_zero R]
+
+/-- extension of `char_zero.cast_injective` to integers. -/
```

Comment:
What makes this an extension? Doesn't `char_zero.cast_injective` already apply to integers?

Reply:
Ah sorry, I misread this.

Your lemma already exists as `int.cast_injective`, as CI is telling you